src/app/series/directives/series-podcast.component.css

```diff
@@ -3,24 +3,9 @@
   width: 250px;
   margin-right: 40px;
 }
-.prx-feed-url input, .feed-url input {
+.prx-feed-url input {
   padding: 10px 8px;
-  color: #939393;
   margin-bottom: 4px;
-}
-.prx-feed-url input {
+  color: #939393;
```

Comment:
Redundant `color: #939393`. This isn't really your doing, but this color on input is (and has been) specified in the global CSS.